content/blog/2023/08/camel4-whatsnew/index.md

```diff
@@ -0,0 +1,185 @@
+        </properties>
+    </bean>
+
+    <route id="my-route">
+        <from uri="direct:start"/>
+        <bean ref="greeter"/>
+        <to uri="mock:finish"/>
+    </route>
+
+</camel>
+```
+
+Then Camel handles the dependency injection, among the `<beans>`. 
+
+You can also use Spring dependency injection (which is more advanced), by inlining Spring `<beans>` tag with the spring namespace.
+See more in the documentation for `camel-xml-io-dsl` module.
+
+There is more work to be done, and we are planning to see if we can also use this to make migration from legacy
+OSGi Blueprint (and Spring XML files, eg `<beans>`) to modern Camel DSL.
+
+### 9) New components
+
+- `camel-aws2-step-functions`: Manage and invoke AWS Step functions
+- `camel-azure-files`: Send and receive files on Azure File Storage
+- `camel-dhis2`: Integrate with DHIS2 (health-level)
+- `camel-opensearch`: Send requests to OpenSearch
+- `camel-parquet-avro`: Parquet Avro serialization and de-serialization
+- `camel-platform-http-main`: Platform HTTP for Camel Main runtime
+- `camel-yaml-io`: YAML DSL route dumper
+
+### 10) Miscellaneous improvements
+
+Camel 4 now requires JUnit 5 for unit tests, with the test components that have -junit5 as suffix.
+
+If you work with XML or JSon payloads then the body can be logged in pretty format with:
+
+    .log("${prettyBody}")
+
+And in XML
+
+    <log message="${prettyBody}"/>
+
+And in YAML
+
+    - log: "${prettyBody}"
+
+
+TODO: Other bits and pieces
+
+### 11) Migrating to Camel 4
+
+We have of course cleaned up the code base, such as removing all deprecated APIs and components. 
+We have also adjusted some APIs in regard to configuring `CamelContext` with custom settings.
+
+In terms of backward compatibility then Camel 4 is mostly compatible for regular Camel applications.
+However, if you are using some of the more advanced features and other plugins in Camel then migration is needed.
+Also, custom components must be migrated and recompiled.
+All details can be seen in the [migration guide](/manual/camel-4-migration-guide.html).
+
+Good luck with your migration if you decide to continue your Camel journey. And for new users to Camel then good luck getting onboard.
+
+### 12) Roadmap for remainder of 2023
+
+We will continue working on Camel 4.x and do non-LTS releases, leading up to the next LTS release by end of this year.
+The major goals for this is Java 21 support, Spring Boot 3.2, and to catch up with newer Quarkus releases.
+
+The following release scheduled, is subject for change.
+
+| Release | Date | Description |
+|---------|------|-------------|
+| 4.1     | Oct  | Non-LTS     |
+| 4.2     | Dec  | LTS         |
+
+An ongoing effort is also to keep stabilizing our CI builds, to ensure commits do not introduce regressions.
+At this moment then the CI builds are occasionally have a few test errors that are related to flaky tests, that
+we keep fixing, to ensure the CI reports are trustworthy.
```

Comment:
I am thinking if we should also comment about the (best-effort) tests we are doing on `Power` and `s390x`. We did receive many contributions from folks interested in those architecture and I think it would be nice to mention that.

Reply:
Yes you are welcome to add some text about that

Reply:
Thanks. Let me try adding something quickly